### PR TITLE
Coding: Develop First Time Issue Section of Contribute Page

### LINKED ISSUE #67: Coding: Develop First Time Issue Section of Contribute Page
## Description
As a developer,
I need the following section of mockup to be developed,
so that I can add as First Time Issue Component [here](https://github.com/anitab-org/anitab-org.github.io/tree/develop/src/Components).

## Mocks
code-first-time-issues

Font Size for heading SOME COOL FIRST TIME ISSUES TO WORK ON: Header Medium(32pt) weight- Medium
Font Size for the description below: body 18pt weight- Light
Font Style- Avenir
Font Color (heading): #103B81
Font Color (Body): #103B81

[Design Link For Development
](https://xd.adobe.com/view/b3ce723d-5bc6-45f5-7035-0a637d919751-c5ff/)

### Update [Required]
- [ ] Follows all the design guidelines as mentioned and is responsive enough on large screen devices.
- [ ] Fetch First Time only issue from GitHub.
- [ ] See all issues on GitHub is a link that should redirect the user to issues page showing all first-time issues available on GitHub.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
3-5 hours

### PR DESCRIPTION
**Pull request by rutvi18**
### Description

Include a summary of the change and relevant motivation/context. List any dependencies that are required for this change.

Fixes #67

### Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
If you are working on a design then add a gif to show the responsiveness of your design.
Please follow [testing guidelines](https://github.com/anitab-org/anitab-org.github.io/wiki/Prepare-testing-GitHub-Pages-for-your-PR) and share the link of deployed site here.

### Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have attached link of deployed site.
- [ ] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by parthsarthiprasad** [comment]
@rutvi18 , thanks for this PR, I was following this issue as well, can you please edit the heading to mention the issue that you are working on and liking it with  "#67" , that'll help reviewers to find out your PR easily as well.Following the template. Thanks.

**Comment by rutvi18** [comment]
@parthsarthiprasad do you mean linking to #67 ?

**Review by prathmeshkhandelwal1 — COMMENTED** [review]
@rutvi18 This PR is giving merge conflict, You should resolve it first. and also plz fill the checklist and types of changes you have done in this PR. You can fill checklist by filling x b/w the brackets `[x]`, And if possible also add ScreenShot of your changes. Feel Free to ask anyone If you stuck anywhere. ;)

File: src/content/contributeIntro.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+          {
+            "par": "Each active repository has an individual stream on Zulip. If you have any questions and not sure how to get started, please ping us on Zulip and we are happy to help!"
+          }
+        ]
+      },
+      {
+          "title":"Ways to contibute",
+          "content":[
+             {
+               "par":"bdjbc"
```
Comment (annabauza) [inline src/content/contributeIntro.json:27]: Random text will not be merged, neither Lorem ipsum.

**Review by annabauza — CHANGES_REQUESTED** [review]

**Comment by rutvi18** [comment]
@tichnas I am closing this PR.

**Pull request closed without merging**